guides/common/modules/proc_migrating_project_or_smartproxy_using_backup.adoc

```diff
@@ -0,0 +1,45 @@
+[id="migrating-{project-context}-or-proxy-using_backup_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL}{nbsp}9
+
+You can migrate your existing {Project} and {SmartProxy} installation from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+
+.Terminology
+Ensure that you understand the following terms:
+
+*Source server*:: The existing {EL}{nbsp}8 system on which {Project} or {SmartProxy} is installed.
+
+*Target server*:: The new {EL}{nbsp}9 system to which {Project} or {SmartProxy} will be migrated.
+
```

Comment:
I don't think you need this.

You're not using "source server" anywhere in the procedure => no need to define it.

Rather than define "target server", you could try using `{EL}{nbsp}9` in the procedure. In the context of this procedure, that would leave no room for doubt about what system you're describing.

In any case, I believe using a `.Terminology` subheading is an unnecessary divergence from the module template.

Reply:
@asteflova the procedures to which I am redirecting refer to source and target, the definition for these in the backups chapter introduction differs from the context in which they should be used here. But I do agree that the divergence should be avoided. Should I add these definition sans the `.Terminology` in the introduction?